### PR TITLE
Fixing some basic bugs

### PR DESCRIPTION
**Pull request by OHopiak**
I've fixed _init.groovy_ script and the Dockerfile, so jobs are running correctly now
In Ansible I added some portability with Docker installation and changed tag _"command"_ for more flexible tags like _"include_tasks"_ or _"docker"_
Removed the security issue with a private ssh key, now it will be added at run time into a Docker volume

### CODE REVIEW

File: terraform/keys.tfvars [hunk 1]
```diff
@@ -0,0 +1,2 @@
+access_key = "XXXXXXXXXXXXXXXXXXX"
```
Comment (diegows) [inline terraform/keys.tfvars:1]: What would you suggest instead of having keys in this file? Is there a better approach?

**Comment by diegows** [comment]
There are conflicts in the PR, could you fix them?

File: terraform/keys.tfvars (hunk 1 above)
Comment (OHopiak) [reply terraform/keys.tfvars:1]: environmental variables like TF_VAR_access_key can be used, but I added the .tfvars file to show which variables should be created

**Comment by OHopiak** [comment]
image
I don't have write permissions to this repository, and these conflicts can't be solved locally

**Comment by diegows** [comment]
@OHopiak could you fix conflict to merge this PR when you have time?

**Pull request merged**